app/models/persistent_volume.rb

```diff
@@ -1,4 +1,22 @@
 class PersistentVolume < ContainerVolume
   acts_as_miq_taggable
+  serialize :capacity, Hash
   delegate :name, :to => :parent, :prefix => true
+
+  IEC_SIZE_SUFFIXES = %w(Ki Mi Gi Ti).freeze
+  def self.parse_iec_number(value)
```

Comment:
@alongoldboim if you want to move this method, it's ok with me (it would be better to be moved to a more generic part of code, some utility gem, etc.) but don't change the code without a proper separate patch that explains the changes.

I don't want this code to be ambiguous in what it returns. When you run an number conversion (e.g. `Integer(...)`) on a value, if it's impossible it fails with `ArgumentError`, let's do the same instead of gracefully fail returning the string.

Catch the exceptions in the parser.